CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+## 2.10.0 / 2019-05-22
+
+* [CHANGE/BUGFIX] API: Encode alert values as string to correctly represent Inf/NaN. #5582
+* [FEATURE] Template expansion: Make external labels available as `$externalLabels` in alert and console template expansion. #5463
```

Comment:
Didn't we also add $params etc. to console templates also?

Reply:
Yes, good catch. I'll add it as as ENHANCEMENT.